android/quest/src/main/java/org/smartregister/fhircore/quest/ui/shared/models/PatientProfileViewData.kt

```diff
@@ -69,6 +71,7 @@ sealed class ProfileViewData(
       val resources = conditions + guardiansRelatedPersonResource + observations
       val resourcesAsBundle = Bundle().apply { resources.map { this.addEntry().resource = it } }
       arrayListOf(*carePlans.toTypedArray(), resourcesAsBundle)
+      arrayListOf(*practitioners.toTypedArray(), resourcesAsBundle)
```

Comment:
Returned list seems to not include careplans from previous line. Maybe merge to return all in a single list